### ISSUE TITLE
Support for MacBook Pro Late 2013 15" (MacBookPro11,2)

### ISSUE THREAD

**Issue by aakkam22** [imbushuo/mac-precision-touchpad#45]
Any update on when this will be supported? The 13" version is supported but not the 15" and it is not listed on the Readme.

MacbookPro11,2

**Comment by imbushuo** [comment]
Report your trackpad's VID(vendor ID) and PID(product ID) in device manager, I will take a look.

**Comment by aakkam22** [comment]
Sorry for the delay, was a little busy. Assuming that the trackpad is listed as "Apple Multi-Touch", here is the info:

![hid and pid](https://user-images.githubusercontent.com/35240600/35125115-1263d204-fc6e-11e7-81fd-917830b31ebb.PNG)

**Comment by imbushuo** [comment]
Will be supported after resolving one bring-up issue(being investigated now).

**Comment by aakkam22** [comment]
Any update?

Let me know if you need access to any hardware.

**Comment by alexmyself** [comment]
Another futur user here.. :)

**Comment by imbushuo** [comment]
@aakkam22 PID 0262 should have been supported as of today. Try latest release?

**Comment by matthewdias** [comment]
works great for me 👌

**Comment by aakkam22** [comment]
Just got the opportunity to test it. It's loads better than Apple's driver, but it's still not quite as good as the macOS driver. One issue with the driver is that I like to have the tip of my thumb resting on the edge of the glass while I track with my index finger and now it thinks I am pinching to zoom because of this. Didn't have this issue with macOS driver nor Apple's stock bootcamp driver. Not sure how difficult it is to address this but I'd imagine it's pretty tricky :( 

Scrolling in some applications like MS Word bring the entire window down or up when scrolling. It's hard to describe but the entire window moves while you scroll.

Despite these issues this is tremendous work you've done and I am bearing in mind that this is under development. You should be commended on your work brother. Good job.

**Comment by imbushuo** [comment]
@aakkam22 Chrome will introduce Precision Touchpad support in coming releases(I've talked with Chrome folks about that), so overall experience on Chrome will be improved soon. I use a Surface Pro 4 as primary device, and these Chrome scrolling problems exist on Surface too.

I've noticed other weird behaviors as you described and working on a fix. Yes, it's tricky. I need to use some mathematical approaches to resolve this.
